kotlin/src/META-INF/kotlin-contents.xml

```diff
@@ -0,0 +1,15 @@
+<idea-plugin>
+    <extensions defaultExtensionNs="com.google.idea.blaze">
+        <SyncPlugin implementation="com.google.idea.blaze.kotlin.sync.BlazeKotlinSyncPlugin"/>
+        <JavaLikeLanguage implementation="com.google.idea.blaze.kotlin.sync.source.BlazeKotlinJavaLikeLanguage"/>
+    </extensions>
+
+    <extensions defaultExtensionNs="com.intellij">
+        <runConfigurationProducer
+                implementation="com.google.idea.blaze.kotlin.run.producers.BlazeKotlinTestClassConfigurationProducer"
+                order="first"/>
```

Comment:
Is there a specific producer that this needs to override? The scala one is also wrong. They can't all be first. You can use `order="before OtherProducer"`

Reply:
Also specify language here?

Reply:
The scala one was for RunLineMarker.

Reply:
Makes sense, I just copied Scala to start with. 

I think it should go before the Java Producer, that should do the trick ? 

The original Kotlin run producers are already suppressed by the java plugin I think.

Reply:
Oh, then it won't be necessary. You can just get rid of order.

Reply:
There isn't a specific producer to override at the moment, this is wrong.  I wrote this producer before I had any clue what I was doing , so it's just about doing the right thing -- in a cause and effect sort of a way. 

Once this changeset goes in and after a bit of a break I will revisit testing. We are already on junit Jupiter (which we are running via the bazel runner) and we want to use a BDD framework for acceptance and some integration testing. 

When I revisit it I hope to address the mapping of test events to tree like test events as represented in a BDD framework. Kotlin Spek and KotlinTest are both run via Jupiter now. So revisiting  will require the Bazel test runner to do the right thing for Jupiter.

@brendandouglas any comments on what would be required to integrate with Jupiter properly -- and to support tree based test models ?

Reply:
@chaoren The Scala and Kotlin producers don't really collide as they are both performing language specific psi introspections and will naturally bail out. I don't think we need to make the ordering more precise for now.

Reply:
removed the order entries and validated.